R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -971,10 +971,15 @@ test_that("spark.randomForest Classification", {
   predictions <- collect(predict(model, data))$prediction
   expect_equal(length(grep("1.0", predictions)), 50)
   expect_equal(length(grep("2.0", predictions)), 50)
+
+  # spark.randomForest classification can work on libsvm data
+  data <- read.df(absoluteSparkPath("data/mllib/sample_multiclass_classification_data.txt"),
+                source = "libsvm")
+  model <- spark.randomForest(data, label ~ features, "classification")
+  expect_equal(summary(model)$numFeatures, 4)
 })
 
-test_that("spark.gbt", {
-  # regression
+test_that("spark.gbt regression", {
```

Comment:
my thought was actually the other way around - I was going to merge the randomForest ones into one tests.
on a related note, I think mllib.R and test_mllib.R are getting too long we might to break them up into more manageable groups. randomForest and gbt classification and regression (R APIs) are similarly enough that I thought it would make more sense to put them into a group.

But, I don't feel strongly about it either way.